app/controllers/users/piv_cac_authentication_setup_controller.rb

```diff
@@ -73,7 +73,6 @@ def user_piv_cac_form
     end
 
     def process_valid_submission
-      flash[:success] = t('notices.piv_cac_configured')
```

Comment:
This removes it during the sign up flow, but if you add a PIV/CAC from the account view there isn't a success message. A quick fix may be to show it unless the user is signing up?

Reply:
Agreed there should be a notification to the user when adding any MFA from the account page but that's currently not happening for the other options either. Will add a story to get that functionality in.

Reply:
I added a method to make it work for TOTP here: https://github.com/18F/identity-idp/pull/3054. You could borrow that if you wanted? There's also some feature specs in there.

I think that phone and backup codes are the other ones that don't have messages and we can open stories for them.

Reply:
Added. It works but I'm not crazy about the logic. Not sure how to make it better either.